journal-week-1.md

```diff
@@ -1,31 +1,9 @@
-#####what did i learn in week 1?
-- met my fellow students
-- git/github
-  - branches/branch management
-  - repositories
-  - fork/clone
-  - push/pull/merge
-    - oh no, merge conflict!
-  - CLI git commands
-  - github for desktop and sourcetree
-  - the woderful world of markdown
-- tools - CLI, node, atom, slack, etc, etc
-- basics of javascript
-    - 4 primitive types (null, Boolean, number, strings)
-    - functions
-      - definition and invocation
-      - input, output, and side effects
-      - local and global variables
-    - mocha/chai testing
-- basics of test-driven development
-- basics of pair programming
+##### What I learned in my first week of FEE at TIY...
 
+The first week of the Front End Engineering cohort at The Iron Yard was a fun filled learning experience. YAY!  Each day of the first week started off with student orientation where we learned some rules, meet some people, talked about parking, and met some more people.
```

Comment:
Thanks for your reflection! It would be great to see you tackle a bigger concept (such as pair-programming).